Domain/Models/User/User.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace Domain.Models.User
+{
+    public class User : IdentityUser<Guid>
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public bool IsDeleted { get; set; } = false;
+        public string PhoneNumber { get; set; }
```

Comment:
Identity user has default phone number property so we do not need to add  this field manually